Caddyfile.pull-preview

```diff
@@ -0,0 +1,25 @@
+{
+    auto_https disable_redirects
+}
+
+{$PULLPREVIEW_URL} {
+    reverse_proxy web:3000
+
+    handle_path /__clerk/* {
+        uri strip_prefix /__clerk
+        reverse_proxy https://frontend-api.clerk.dev {
```

Comment:
Maybe, consider parameterizing the URL to make it easier to change in the future. The same as is done for https://{$PULLPREVIEW_PUBLIC_DNS}/__clerk